conda_forge_tick/update_recipe/version.py

```diff
@@ -379,7 +379,9 @@ def _try_to_update_version(cmeta: Any, src: str, hash_type: str):
     return updated_version, errors
 
 
-def update_version(raw_meta_yaml, version, hash_type="sha256"):
```

Comment:
The API changes in this file are going to break the admin commands here: https://github.com/conda-forge/webservices-dispatch-action/blob/main/webservices_dispatch_action/version_updater.py

OFC I wrote that integration with no versioning of the bot code base or expectations of API statbility, so the breakage is clearly my fault! 

That all being said, I think at this point we need a versioning policy for the bot code base and we need to distribute it properly. 

I've had some discussion in the past and calver was generally agreed to be the least work given the fluid nature of the bot's APIs. We will need to automate updates of services that depend on the bot's version as well.

cc @jaimergp @ytausch for viz

xref: https://github.com/regro/cf-scripts/issues/2894

Reply:
I'd honestly prefer no API changes and this is the approach I've taken in the past. However, it is very hard for me to keep this approach going given how many PRs are being made on the bot.

Reply:
I appreciate the difficulty here, and are happy to compromise on nice function names in order to avoid breakage.

I will rename this function again, and try my best, to avoid renaming others that are not private.

If you find something that I missed, please let me know.